lib/leaf.js

```diff
@@ -24,6 +24,20 @@ var Leaf = createClass({
             prefix: ''
         };
     },
+    childrenRef:{},
+    onExpandCollpase: function(expanded) {
```

Comment:
consider adding verbose comment here about the usage of `childrenRef`
`// will store all root's childern in order to expand/collapse all`